Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -877,6 +886,9 @@ extension HomepageViewController: Notifiable {
                     .WallpaperDidChange:
                 self.reloadView()
 
+            // Ecosia: Seed Counter Experiment
+            case UserDefaultsSeedProgressManager.levelUpNotification:
+                SeedCounterNTPExperiment.trackSeedLevellingUp()
```

Comment:
Since for now we just track Analytics with this static method, couldn't this be directly handled inside `UserDefaultsSeedProgressManager`? I guess this would simplify and remove the the need for the notification.

Reply:
Replied above. 
Main topic: separation of concerns. I'm happy to discuss that ofc 💪

Reply:
Got it, thanks for explaining, to be honest I'd still be personally a bit on the fence since there is quite some logic required for the HomepageViewController notification handling that could be removed. But I do like the separation of concerns too, so no blockers from my side 🙌